packages/polaris-test/server-without-typeorm/schema/resolvers.ts

```diff
@@ -0,0 +1,245 @@
+import {
+    DeleteResult,
+    getPolarisConnectionManager,
+    Like,
+    PaginatedResolver,
+    PolarisError,
+    PolarisGraphQLContext,
+} from '@enigmatis/polaris-core';
+import { PubSub } from 'apollo-server-express';
+import { TestContext } from '../../shared-resources/context/test-context';
+import { Author } from '../../shared-resources/entities/author';
+import { Book } from '../../shared-resources/entities/book';
+import { polarisGraphQLLogger } from '../../shared-resources/logger';
+import { Pool, PoolClient } from 'pg';
+
+const pubsub = new PubSub();
+const BOOK_UPDATED = 'BOOK_UPDATED';
+
+export const resolvers = {
```

Comment:
use resolvers of test server and override only paginated books resolver, instead of copying file